src/core/Track.cpp

```diff
@@ -2150,10 +2150,10 @@ void Track::loadSettings( const QDomElement & element )
 		node = node.nextSibling();
 	}
 
-	if( element.attribute( "height" ).toInt() >= MINIMAL_TRACK_HEIGHT &&
-		element.attribute( "height" ).toInt() <= DEFAULT_TRACK_HEIGHT )	// workaround for #3585927, tobydox/2012-11-11
+	int storedHeight = element.attribute( "height" ).toInt();
+	if( storedHeight >= MINIMAL_TRACK_HEIGHT )
```

Comment:
`if( storedHeight >= MINIMAL_TRACK_HEIGHT && storedHeight < 400 )`

This works as a quick dirty fix.